src/js/ctrl/demonstrations.ts

```diff
@@ -1296,20 +1325,27 @@ export function cleanSpecForPreview(sceneSpec, width: number, height: number, gr
     return markSpec;
   });
 
-  return addBaseSignalsForPreview(sceneUpdated, groupName, interactionId);
+  return addBaseSignalsForPreview(sceneUpdated, groupName, interactionId, isHistoryItem);
```

Comment:
same idea as `cleanSpecForPreview`. you don't need to be adding base interaction signals to non-interactive history previews